api.go

```diff
@@ -75,44 +57,6 @@ func MakeMinerFromRaw(
 	return ascertain.MakeLicenseMinerFromRaw(minimums, reader)
 }
 
-// Locker is a type alias for ascertain.Locker
-type Locker = ascertain.Locker
-
-// MakeLocker creates a Locker
-func MakeLocker() Locker {
-	return ascertain.MakeDependencyLocker()
-}
-
-// LicenseLockManager is a file manager for the license.lock file.
-// Lock() will create a new file or merge with an existing.
-// Read() will return a list of DependencyLocks, if a file exists.
-type LicenseLockManager interface {
-	Lock(...DependencyLock) error
-	Read() ([]DependencyLock, error)
-}
-
-// MakeLicenseLockManager creates a LicenseLockManager with specified directory
-func MakeLicenseLockManager(
-	dir string,
-) LicenseLockManager {
-	return ascertain.MakeLicenseLock(dir)
-}
-
-// MakeLicenseLockFromRaw creates a LicenseLockManager from specified parameters
-func MakeLicenseLockFromRaw(
-	locker Locker,
-	path string,
-	printer prettyprinter.Printer,
-	reader *reader.LimitedReader,
-) LicenseLockManager {
-	return ascertain.MakeLicenseLockFromRaw(
-		locker,
-		path,
-		printer,
-		reader,
-	)
-}
-
```

Comment:
This code essentially just produces a json file of verified licenses. Only, some of them are potentially invalid because it writes the "locks" (i.e., licenses) to the file _before_ actually auditing them. Nothing about this file is actually locked and the code wasn't correct so I removed the notion of a "license locker"